bots/twitch/phantombot/egg-phantom-bot.json

```diff
@@ -1,24 +1,28 @@
 {
     "_comment": "DO NOT EDIT: FILE GENERATED AUTOMATICALLY BY PTERODACTYL PANEL - PTERODACTYL.IO",
     "meta": {
-        "version": "PTDL_v1"
+        "version": "PTDL_v1",
+        "update_url": null
     },
-    "exported_at": "2020-12-16T19:18:24+02:00",
+    "exported_at": "2021-09-09T12:05:16+02:00",
     "name": "PhantomBot",
-    "author": "mail@wuffy.eu",
```

Comment:
Please do not change the original author of the egg.

Reply:
Sure is back to Normal ^^